langchain4j-oracle/src/test/java/dev/langchain4j/store/embedding/oracle/CommonTestOperations.java

```diff
@@ -0,0 +1,47 @@
+package dev.langchain4j.store.embedding.oracle;
+
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import oracle.jdbc.datasource.OracleDataSource;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+
+/**
+ * A collection of operations which are shared by tests in this package.
+ */
+final class CommonTestOperations {
+
+    /**
+     * Model used to generate embeddings for this test. The all-MiniLM-L6-v2 model is chosen for consistency with other
+     * implementations of EmbeddingStoreIT.
+     */
+    private static final EmbeddingModel EMBEDDING_MODEL = new AllMiniLmL6V2QuantizedEmbeddingModel();
+
+    private CommonTestOperations() {}
+
+    private static final DataSource DATA_SOURCE;
+    static {
+        try {
```

Comment:
As the comment in the pom, shouldn't we provide a PooledDataSource (PoolDataSourceFactory.getPoolDataSource();)

Reply:
We want to avoid the introduction of new dependencies.